test/e2e/data/infrastructure-metal3/bases/cluster-with-kcp.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: cluster.x-k8s.io/${CAPI_VERSION}
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: ${NAMESPACE}
```

Comment:
I think we can remove the namespace here and put it in the `kustomize.yaml` file instead.

Reply:
As a result the namespace will be still a parameter do not see the reason for that

Reply:
It is just simpler. One place instead of many. Now we need to have it for the Cluster, Metal3Cluster, KubeadmControlPlane and Metal3MachineTemplates. If we put it in `kustomize.yaml` it is only in one place and applies to all of them I think.